### ISSUE TITLE
A few broken links in experimenter docs

### ISSUE THREAD

**Issue by pdehaan** [mozilla/experimenter#5940]
Not a very good script (and takes about 2 minutes to scrape 1200+ links in experimenter.info site, which might make it annoying to run in CI).

TLDR: Found 1207 links (and 20 broken links).
But it looks like 10 of the 20 links are the same relative `feature-variables-and-me` link in the nav bar which should be an absolute `/feature-variables-and-me` link).

The [HTTP_429] errors might just be my getting rate limited by STMO; [429 Too Many Requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429).

And the "/my-flying-nimbus" might be an expected error of sorts since that looks like a sample/fake URL.

Lastly there is a https://settings-writer.stage.mozaws.net link which probably requires VPN, so I think the link checker just tapped out on figuring that link out. There might be a way to configure the script to ignore certain links.

```sh
> experimenter-link-checker@1.0.0 start /private/tmp/experimenter-link-checker
> time node link-check

https://experimenter.info/contributing
[HTTP_404] https://github.com/mozilla/experimenter-docs/edit/my-flying-nimbus/sidebars.js
    html: <a href="https://github.com/mozilla/experimenter-docs/edit/my-flying-nimbus/sidebars.js" target="_blank" rel="noopener noreferrer">this would be the link</a>

https://experimenter.info/console/custom-audiences
[HTTP_404] https://experimenter.info/console/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/console/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/console/preview
[HTTP_404] https://experimenter.info/console/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/console/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/jetstream/jetstream
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/jetstream/metrics
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/jetstream/statistics
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/jetstream/outcomes
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/jetstream/configuration
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

https://experimenter.info/jetstream/data-products
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

[HTTP_429] https://sql.telemetry.mozilla.org/dashboard/jetstream-errors?p_experiment=%25
    html: <a href="https://sql.telemetry.mozilla.org/dashboard/jetstream-errors?p_experiment=%25" target="_blank" rel="noopener noreferrer">Jetstream error dashboard</a>

[HTTP_429] https://sql.telemetry.mozilla.org/alerts/81
    html: <a href="https://sql.telemetry.mozilla.org/alerts/81" target="_blank" rel="noopener noreferrer">setting up alerts</a>

[HTTP_429] https://sql.telemetry.mozilla.org/dashboard/jetstream-cost
    html: <a href="https://sql.telemetry.mozilla.org/dashboard/jetstream-cost" target="_blank" rel="noopener noreferrer">jetstream cost monitoring dashboard</a>

[HTTP_429] https://sql.telemetry.mozilla.org/alerts/91
    html: <a href="https://sql.telemetry.mozilla.org/alerts/91" target="_blank" rel="noopener noreferrer">alerts set up</a>

https://experimenter.info/jetstream/operations
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

[HTTP_404] https://github.com/argoproj/argo/blob/master/docs/quick-start.md
    html: <a href="https://github.com/argoproj/argo/blob/master/docs/quick-start.md" target="_blank" rel="noopener noreferrer">Argo's installation guide</a>

https://experimenter.info/jetstream/troubleshooting
[HTTP_404] https://experimenter.info/jetstream/feature-variables-and-me
    html: <a class="menu__link" tabindex="-1" href="/jetstream/feature-variables-and-me">Feature Variables API</a>

[HTTP_429] https://sql.telemetry.mozilla.org/dashboard/jetstream-errors?p_experiment=%25
    html: <a href="https://sql.telemetry.mozilla.org/dashboard/jetstream-errors?p_experiment=%25" target="_blank" rel="noopener noreferrer">Jetstream error dashboard</a>

[HTTP_429] https://sql.telemetry.mozilla.org/alerts/81
    html: <a href="https://sql.telemetry.mozilla.org/alerts/81" target="_blank" rel="noopener noreferrer">Redash alert configuration</a>

https://experimenter.info/data-scientists
[HTTP_404] https://docs.telemetry.mozilla.org/tools/experiments.html
    html: <a href="https://docs.telemetry.mozilla.org/tools/experiments.html" target="_blank" rel="noopener noreferrer">datasets used in experimentation</a>

https://experimenter.info/desktop-rollouts
[BLC_UNKNOWN] https://settings-writer.stage.mozaws.net/v1/admin/#/buckets/main-workspace/collections/nimbus-desktop-defaults/records
    html: <a href="https://settings-writer.stage.mozaws.net/v1/admin/#/buckets/main-workspace/collections/nimbus-desktop-defaults/records" target="_blank" rel="noopener noreferrer">remote settings collection</a>

Found 1207 links (and 20 broken links).


real    2m8.366s
user    0m3.634s
sys     0m0.379s
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1484)

**Comment by pdehaan** [comment]
<details>
<summary>Clicko expando!</summary>

```js
const { SiteChecker } = require("broken-link-checker");

const siteUrl = "https://experimenter.info";
const options = {};
const customData = { errors: new Map(), linkCount: 0, brokenLinkCount: 0 };

var siteChecker = new SiteChecker(options, {
  robots(robots, customData) {
    // console.log("robots:", {robots});
  },
  html(tree, robots, response, pageUrl, customData) {},
  junk(result, customData) {},
  link(result, customData) {
    customData.linkCount += 1;
    if (result.broken) {
      customData.errors.set(result.url.resolved, {
        reason: result.brokenReason,
        page: result.base.resolved,
        resolved: result.url.resolved,
        html: `${result.html.tag}${result.html.text}</${result.html.tagName}>`
      });
    }
  },
  page(error, pageUrl, customData) {
    if (customData.errors.size) {
      console.error(pageUrl);
      customData.brokenLinkCount += customData.errors.size;
      const res = Array.from(customData.errors.values());
      for (const err of res) {
        console.error(`[${err.reason}] ${err.resolved}`);
        console.error(`    html: ${err.html}\n`);
      }
      customData.errors.clear();
    }
  },
  site(error, siteUrl, customData) {},
  end() {
    console.log(`Found ${customData.linkCount} links (and ${customData.brokenLinkCount} broken links).\n`);
    if (customData.brokenLinkCount) {
      process.exitCode = 1;
    }
  },
});

siteChecker.enqueue(siteUrl, customData);
```
</details>

**Comment by pdehaan** [comment]
So, if we have 20 broken links, but 6 are STMO, 1 is Remote Settings, and 10 are the "Feature Variables API" link pathing issue, that leaves (maths) 2-3 possibly legit broken links worthy of investigation:

```
https://experimenter.info/contributing
[HTTP_404] https://github.com/mozilla/experimenter-docs/edit/my-flying-nimbus/sidebars.js
    html: <a href="https://github.com/mozilla/experimenter-docs/edit/my-flying-nimbus/sidebars.js" target="_blank" rel="noopener noreferrer">this would be the link</a>

https://experimenter.info/jetstream/operations
[HTTP_404] https://github.com/argoproj/argo/blob/master/docs/quick-start.md
    html: <a href="https://github.com/argoproj/argo/blob/master/docs/quick-start.md" target="_blank" rel="noopener noreferrer">Argo's installation guide</a>

https://experimenter.info/data-scientists
[HTTP_404] https://docs.telemetry.mozilla.org/tools/experiments.html
    html: <a href="https://docs.telemetry.mozilla.org/tools/experiments.html" target="_blank" rel="noopener noreferrer">datasets used in experimentation</a>
```

**Comment by jaredlockhart** [comment]
I'd like to find a better way to track docs tickets but leaving this open for now 👍

**Comment by jaredlockhart** [comment]
We're gonna review the experimenter docs in more detail so we can review links when we get to that.
